mono/utils/mono-context.h

```diff
@@ -24,6 +24,8 @@
 
 #if defined(__APPLE__)
 typedef struct __darwin_xmm_reg MonoContextSimdReg;
+#elif defined(__linux__)
+typedef struct _libc_xmmreg MonoContextSimdReg;
```

Comment:
I was under the impression that `_libc_xmmreg` is only defined on x86-64.

Reply:
@evincarofautumn That is correct, it wasn't used in the added code. I made the typedefs explicit relative to the target. I can't see any way of fetching the xmm regs on x86 on linux.